playfab-docs/features/data/dataconnections/data-connection-adx.md

```diff
@@ -55,16 +55,14 @@ Once your Azure AD app is listed as a “Database Admin”, you can move on to c
 | Cluster URI  | The ADX cluster URI.|
 | Database | The database name. The PlayFab ingestion service will create a table in this database and ingest the data to that table. |
 | Table | The name of the table in the database that PlayFab will ingest the data into. If the table does not exist, then the ingestion service will create the table. |
-| Application ID | The Azure AD App ID. |
-| Tenant ID | The Azure AD Tenant ID. |
 
-![Screenshot of configuring and ADX data connection](media/configure-new-data-connection.png "ADX data connection configuration") 
+![Screenshot of configuring and ADX data connection](media/Config-ADX-Data-Connections.png "ADX data connection configuration") 
 
 > [!Note]
 > We recommend using a table name that does not exist. If you use an existing table and the schema is not what PlayFab is expecting, then the schema will be modified. 
 
-### Step 3: Create and Share a Client Secret 
+### Step 3: Ensure that you have granted the Azure Active Directory app access to your Azure Data Explorer cluster before proceeding. 
 
-Return to your Azure AD App’s “App Registration” to create a client secret. You can follow the steps here to create a new client secret for your app: [How to Create a New Application Secret](/azure/active-directory/develop/howto-create-service-principal-portal#option-2-create-a-new-application-secret) 
+Follow the commands as shown in the setup window once you set up your Data connection. 
```

Comment:
It would be better to list what they need to do briefly in the commands, instead of relying on an image. It's also more accessible.
Also, couldn't tell what I needed to do based on the image.

Reply:
I'm expecting a bullet list of things i need to configure or set when i'm in the Azure AD app.

So perhaps start with:

Return to your Azure AD App (add a URL). Navigate to the App Registration page.
ETC.

Be as clear as you can be.